### PR TITLE
bce-iot-9121: modify docs of new setup method

### PR DESCRIPTION
**Pull request by chensheng0**

### CODE REVIEW

File: doc/us-en/setup/Quick-Install.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Quick Install OpenEdge
+
+Compared to the previous version which using manual download installation, the latest version (version 0.1.5) OpenEdge supports package manager installation. With package manager, users can quickly install OpenEdge by simply typing a few commands at the terminal.
```
Comment (kuailehaibin) [inline doc/us-en/setup/Quick-Install.md]: Compared to the previous version which using manually download installation, the latest version (0.1.5) of OpenEdge supports command line installation. With this method, users can quickly install OpenEdge by simply typing a few commands with the terminal.

File: doc/us-en/setup/Quick-Install.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+- executing the command `sudo systemctl status openedge` to check whether `openedge` is running, as shown below. Otherwise, `openedge` fails to start.
+
+![OpenEdge](../../images/setup/openedge-systemctl-status.png)
+
+- executing the command `docker stats` to view the running status of docker containers. Since the main program `openedge` will first pull required images from mirror repository, users needs to wait 2~5 minutes to execute this command. Take the default configurations in above step for example, the running status of containers are as shown following after pulled. If some containers are missing, it says they failed to start.
+
+![docker stats](../../images/setup/docker-stats.png)
+
+- Under the condition of two above failures, you need to view the log to check more of main program. Log files are stored in `/usr/local/var/log/openedge/openedge.log` by default. For errors in the log, users can refer to [FAQ](../FAQ.md). If necessary, just [Submit an issue](https://github.com/baidu/openedge/issues) for help.
```
Comment (kuailehaibin) [inline doc/us-en/setup/Quick-Install.md]: Under the condition of two above failures, you need to view the log of main program. And the log file which is stored in `/usr/local/var/log/openedge/openedge.log` by default. Once found errors in the log file, users can refer to [FAQ](../FAQ.md). If necessary, just [Submit an issue](https://github.com/baidu/openedge/issues) for help.

File: doc/zh-cn/setup/Quick-Install.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# 快速安装 OpenEdge
+
+相较于之前版本的手动下载安装方式，最新版本(0.1.5 版本) OpenEdge 支持包管理器的安装方式。通过包管理器方式，用户可以在终端简单输入几条命令，快速安装 OpenEdge。
+
+OpenEdge 包管理器目前支持的系统有: Ubuntu Server 16.04、Ubuntu Server 18.04、Debian9、CentOS7、Raspbian-stretch，支持的平台有 X64、i386、ARM32 和 ARM64。
```
Comment (kuailehaibin) [inline doc/zh-cn/setup/Quick-Install.md]: ubuntu非服务器版不支持？参考英文的改一下

File: doc/zh-cn/setup/Quick-Install.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+## 安装容器运行时
+
+在 **docker** 容器模式下，OpenEdge 依赖于 docker 容器运行时。如果用户机器尚未安装 docker 容器运行时，可通过以下命令安装 docker 的最新版本（适用于类 Linux 系统）:
+
+```shell
+curl -sSL https://get.docker.com | sh
+```
+
+安装结束后，查看 docker 的版本号:
```
Comment (kuailehaibin) [inline doc/zh-cn/setup/Quick-Install.md]: 一旦安装完成，用户可以通过下面的命令查看 docker 的版本号：

File: doc/us-en/setup/Quick-Install.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+# Quick Install OpenEdge
+
+Compared to the previous version which using manual download installation, OpenEdge supports package manager installation in the latest version(0.1.5). With this method, users can quickly install OpenEdge by simply typing a few commands at terminal.
+
+Installation packages are provided for Ubuntu Server 16.04, Ubuntu Server 18.04, Debian9, CentOS7, and Raspbian-stretch currently. The supported platforms are X64, i386, ARM32, and ARM64.
```
[the remark this answers is not in the document]
Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: Installation packages are provided for Ubuntu16.04, Ubuntu18.04, Debian9., CentOS7., and Raspbian-stretch currently. The supported platforms are amd64, i386, armv7l, and arm64.

File: doc/zh-cn/setup/Quick-Install.md (hunk 4 above)
Comment (chensheng0) [reply doc/zh-cn/setup/Quick-Install.md]: 安装结束后，用户可以查看 docker 的版本号:

File: doc/us-en/setup/Build-from-Source.md [hunk 6]
```diff
@@ -109,9 +83,9 @@ go env  # View Go's environment variables
 go version # View Go's version
 ```
 
-_**NOTE**: OpenEdge requires that the compiled version of Go be above 1.10.0 ._
+**NOTE**: OpenEdge requires that the compiled version of Go should be above 1.10.0.
 
-#### Install Docker
+#### Install the container runtime
```
Comment (kuailehaibin) [inline doc/us-en/setup/Build-from-Source.md]: Install the container

File: doc/us-en/setup/Build-from-Source.md [hunk 7]
```diff
@@ -20,57 +20,31 @@ go env  # View Go's environment variables
...
-**CentOS**
-
-Command:
-
-```shell
-yum install docker
-```
-
-**NOTE**: After the docker installation is complete, use the following command to view the installed version of docker.
+After the installation is complete, users can check the version of `docker`:
```
Comment (kuailehaibin) [inline doc/us-en/setup/Build-from-Source.md]: Once the installation is complete, users can check the version of `docker`:

File: doc/us-en/setup/Quick-Install.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
+# Quick Install OpenEdge
+
+Compared to manual download software in previous version, it supports installing `OpenEdge` through package manager in latest version(0.1.5). With this method, users can quickly install `OpenEdge` by simply typing a few commands at terminal.
```
Comment (kuailehaibin) [inline doc/us-en/setup/Quick-Install.md]: package manager 不是说了这种表述是错误的吗？用 package management

File: doc/us-en/setup/Quick-Install.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+Installation packages are provided for Ubuntu16.04, Ubuntu18.04, Debian9., CentOS7., and Raspbian-stretch currently. The supported platforms are amd64, i386, armv7l, and arm64.
```
Comment (kuailehaibin) [inline doc/us-en/setup/Quick-Install.md]: Debian9.X，CentOS7.X

File: doc/us-en/setup/Quick-Install.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+OpenEdge supports two running modes: **docker** container mode and **native** process mode. This document will be described in **docker** container mode.
+
+## Install the container runtime
```
Comment (kuailehaibin) [inline doc/us-en/setup/Quick-Install.md]: Install the container

File: doc/zh-cn/setup/Quick-Install.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+- 在终端中命令 `sudo systemctl status openedge` 来查看 `openedge` 是否正常运行。正常如下图所示，否则说明主程序 `openedge` 启动失败；
+
+![OpenEdge](../../images/setup/openedge-systemctl-status.png)
+
+- 在终端中执行命令 `docker stats` 查看当前 docker 中容器的运行状态。由于主程序 `openedge` 会先到镜像仓库拉取需要的镜像，用户需要等待 2~5 分钟执行此条命令。以上一步中导入的默认配置为例，待主程序拉取完成后，容器的运行状态如下图所示。如果用户本地的镜像与下述不一致，说明模块启动失败；
+
+![当前运行 docker 容器查询](../../images/setup/docker-stats.png)
+
+- 针对上述两种失败情况，用户需要查看主程序日志来了解具体的错误情况。主程序日志的默认存放位置为 `/usr/local/var/log/openedge/openedge.log`。针对日志中出现的错误，用户可先参考 [常见问题](../FAQ.md) 进行解决。必要时可以直接 [提交 Issue](https://github.com/baidu/openedge/issues)。
```
Comment (kuailehaibin) [inline doc/zh-cn/setup/Quick-Install.md]: 按 丹峰 会上说的，加一下 openedge stop 的命令 `systemctl stop openedge`，再加一段描述

File: doc/us-en/setup/Build-from-Source.md [hunk 12]
```diff
@@ -20,57 +20,31 @@ go env  # View Go's environment variables
 go version # View Go's version
 ```
 
-_**NOTE**: OpenEdge requires that the compiled version of Go be above 1.10.0._
+**NOTE**: OpenEdge requires that the compiled version of Go should be above 1.10.0.
 
-#### Install Docker
+#### Install the container runtime
```
Comment (kuailehaibin) [inline doc/us-en/setup/Build-from-Source.md]: Install the container

File: doc/us-en/setup/Quick-Install.md (hunk 8 above)
Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: wiki 上都用 package manager

Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: https://en.wikipedia.org/wiki/Package_manager

File: doc/us-en/setup/Quick-Install.md (hunk 9 above)
Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: 直接 Debian9， CentOS7

File: doc/zh-cn/setup/Quick-Install.md (hunk 11 above)
Comment (chensheng0) [reply doc/zh-cn/setup/Quick-Install.md]: 加了

File: doc/us-en/setup/Build-from-Source.md (hunk 12 above)
Comment (chensheng0) [reply doc/us-en/setup/Build-from-Source.md]: 还是 Install the container runtime ，参考外文的结论

File: doc/zh-cn/setup/Quick-Install.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+```shell
+curl -sSL http://download.openedge.tech/install.sh | sudo -E bash -
+```
+
+执行完毕后，OpenEdge 将会被安装到 `/usr/local` 目录下。
+
+最新版本的 OpenEdge 使用 Systemd 守护，用户可使用以下命令启动 OpenEdge:
+
+```shell
+sudo systemctl start openedge
```
Comment (kuailehaibin) [inline doc/zh-cn/setup/Quick-Install.md]: 确认下是否需要 sudo 权限启动、停止

Comment (chensheng0) [reply doc/zh-cn/setup/Quick-Install.md]: 确认了，是需要 sudo 的

File: doc/us-en/setup/Quick-Install.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+OpenEdge relies on docker container runtime in **docker** container mode. Users can install docker (for Linux-like systems) with the following command if it's not installed yet:
+
+```shell
+curl -sSL https://get.docker.com | sh
+```
+
+Users can check the version of `docker` after installation is complete,:
```
[the remark this answers is not in the document]
Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: Users can check the version of `docker` when the installation is completed:

File: doc/us-en/setup/Quick-Install.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+Compared to the previous version which using manual download installation, OpenEdge supports package manager installation in the latest version(0.1.5). With this method, users can quickly install OpenEdge by simply typing a few commands at terminal.
+
+Installation packages are provided for Ubuntu Server 16.04, Ubuntu Server 18.04, Debian9, CentOS7, and Raspbian-stretch currently. The supported platforms are X64, i386, ARM32, and ARM64.
+
+OpenEdge supports two running modes: **docker** container mode and **native** process mode. This article's quick installation is based on **docker** container mode.
+
+## Install the container runtime
+
+OpenEdge relies on docker container runtime in **docker** container mode. Users can install docker (for Linux-like systems) with the following command if it's not installed yet:
```
[the remark this answers is not in the document]
Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: docker container runtime， 参考外文的

File: doc/us-en/setup/Quick-Install.md (hunk 1 above)
Comment (chensheng0) [reply doc/us-en/setup/Quick-Install.md]: Compared to manual download software in previous version, it supports installing OpenEdge through package manager in latest version(0.1.5). With this method, users can quickly install OpenEdge by simply typing a few commands at terminal.

File: doc/us-en/setup/Build-from-Source.md (hunk 7 above)
Comment (chensheng0) [reply doc/us-en/setup/Build-from-Source.md]: View the version of installed docker:

**Pull request merged**